pcsx2/IopMem.cpp

```diff
@@ -490,3 +490,14 @@ void __fastcall iopMemWrite32(u32 mem, u32 value)
 		}
 	}
 }
+
+std::string iopMemReadString(u32 mem, int maxlen)
```

Comment:
That's actually intended specifically for strings which aren't NUL terminated, it's not just for sanity.